docs/website/src/site/index.md

```diff
@@ -266,7 +266,7 @@ Guidelines
 
 <section class="ds-call-to-action">
 
-<svg width="296" height="308" viewBox="0 0 296 308" fill="none" xmlns="http://www.w3.org/2000/svg" aria-labelledby="ctaTitle ctaDesc" role="img">
+<svg width="296" height="308" viewBox="0 0 296 308" fill="none" preserveAspectRatio="none" xmlns="http://www.w3.org/2000/svg" aria-labelledby="ctaTitle ctaDesc" role="img">
```

Comment:
why did you add this? if the width and height are set explicitly it shouldn't be necessary, and also i don't think `none` would be the best value, since that would cause the image to distort.

Reply:
yes, it's not necessary for now, I was just trying different solutions for Safari and I left it by chance. :)